utils/build.ps1

```diff
@@ -402,96 +407,98 @@ function Invoke-VsDevShell($Arch) {
   }
 }
 
-function Ensure-WindowsSDK {
-  # Assume we always have a default Windows SDK version available
-  if (-not $WinSDKVersion) { return }
-
-  # Check whether VsDevShell can already resolve the requested Windows SDK version
-  try {
-    Isolate-EnvVars { Invoke-VsDevShell $HostArch }
-    return
-  } catch {}
-
-  Write-Output "Windows SDK $WinSDKVersion not found. Downloading from nuget..."
+function Fetch-Dependencies {
+  $ProgressPreference = "SilentlyContinue"
 
-  # Assume the requested Windows SDK is not available and we need to download it
-  # Install the base nuget package that contains header files
-  $WinSDKBasePackageName = "Microsoft.Windows.SDK.CPP"
-  Invoke-Program nuget install $WinSDKBasePackageName -Version $WinSDKVersion -OutputDirectory $NugetRoot
+  $WebClient = New-Object Net.WebClient
+  $WiXVersion = 4.0.1
+  $WiXURL = https://www.nuget.org/api/v2/package/wix/$WiXVersion
+  $WiXHash = "756AD3115F0CE808313266F4E401C0F520D319211DE0B9D8D7E7697020E0C461"
 
-  # Export to script scope so Invoke-VsDevShell can read it
-  $script:CustomWinSDKRoot = "$NugetRoot\$WinSDKBasePackageName.$WinSDKVersion\c"
-
-  # Install each required arch-specific package and move the files under the base /lib directory
-  $WinSDKArchs = $SDKArchs.Clone()
-  if (-not ($HostArch -in $WinSDKArchs)) {
-    $WinSDKArchs += $HostArch
-  }
+  if (-not (Test-Path $BinaryCache\WiX-$WiXVersion.zip)) {
+    Write-Output "WiX not found. Downloading from nuget.org ..."
+    if ($ToBatch) {
+      Write-Output "curl.exe -sL $WiXURL -o $BinaryCache\WiX-$WiXVersion.zip"
```

Comment:
Do we want to add the expand archive part to the ToBatch case?

Reply:
It is a PowerShell builtin, so I don't see how to do that :(

Reply:
We probably could rewrite archive expansion with `tar.exe`, as it is part of Windows now (and git provides tar too, which makes things a bit complicated, as they are not 100% interchangeable). It will be just another `Invoke-Program` then.

Reply:
Yeah, I am not sure if that is available on the CI builders.  We should do that as a follow up.